tests/unit/models/test_input.py

```diff
@@ -554,3 +655,111 @@ def test_field_accepts_none(self) -> None:
         model = self.TestModel(arch_filter=None)
         assert model.arch_filter.filters == set()
         assert model.arch_filter.is_all is True
+
+
+class TestPipBinaryFilters:
+    def test_default_values(self) -> None:
```

Comment:
Here we test that pydantic sets default values to the values that we asked it to use. Which means that the test is for pydantic. Does it provide value to Hermeto?